miden/src/cli/prove.rs

```diff
@@ -38,14 +39,25 @@ pub struct ProveCmd {
     /// Paths to .masl library files
     #[structopt(short = "l", long = "libraries", parse(from_os_str))]
     library_paths: Vec<PathBuf>,
+
+    /// Maximum number of cycles a program is allowed to consume
+    #[structopt(short = "m", long = "max-cycles")]
+    max_cycles: Option<u32>,
+
+    /// Expected number of cycles
+    #[structopt(short = "e", long = "exp-cycles", default_value = "64")]
+    expected_cycles: u32,
```

Comment:
Let's change the description to "Number of cycles the program is expected to consume".

Also, let's arrange all options here (and in the `run.rs`) alphabetically.